### ISSUE TITLE
tainio-mania.online: ads

### ISSUE THREAD

**Issue by fanis2022** [uBlockOrigin/uAssets#15611]
### Prerequisites

- [X] I read and understood the [policy as to what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets)
- [X] I verified that the issue has not already been reported (use [this button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out)
- [X] I forced an update of my filter lists (**how to do this:** [click _"Purge all caches"_](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches), then [click _"Update now"_](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now))
- [X] I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/585534/91296476-ab9d8c00-e76a-11ea-9fb9-87d6861bdf73.png), or I verified that the issue is not caused by removing any of the default lists
- [X] I did not enable external filter lists, or I verified that the issue still occurs without enabling external filter lists
- [X] I do not have custom filters/rules, or I verified that the issue still occurs without custom filters/rules
- [X] I am not using [another content blocker along uBO](https://twitter.com/gorhill/status/1033706103782170625)
- [X] I verified the issue is not caused by the browser built-in blocker, or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through VPN)
- [X] I did not answer truthfully to **all** the above checkpoints

### URL address of the web page

`https://tainio-mania.online/`

### Category

ads

### Describe the issue

annoying popups

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.45.0
Firefox: 106
filterset (summary): 
  network: 89510
  cosmetic: 25
  scriptlet: 16667
  html: 795
listset (total-discarded, last-updated): 
  default: 
    user-filters: 28-2, never
    ublock-filters: 31897-8193, 1d.12h.2m
    ublock-badware: 3962-634, 1d.12h.1m
    ublock-privacy: 285-10, 1d.6h.18m
    ublock-abuse: 76-1, 1d.6h.18m
    ublock-unbreak: 1906-250, 1d.12h
    ublock-quick-fixes: 542-191, 5h.51m
    easylist: 62820-29834, 5d.46m
    easyprivacy: 30779-1725, 5d.45m
    urlhaus-1: 11582-0, 5h.51m
    plowe-0: 3669-3, 7d.6h.32m
    GRC-0: 1002-548, 1d.10h.24m
filterset (user): [array of 28 redacted]
trustedset: 
  added: [array of 2 redacted]
  removed: 
    moz-extension-scheme
switchRuleset: 
  added: [array of 1 redacted]
modifiedUserSettings: 
  ignoreGenericCosmeticFilters: true
  parseAllABPHideFilters: false
  suspendUntilListsAreLoaded: false
modifiedHiddenSettings: [none]
supportStats: 
  allReadyAfter: 324 ms (selfie)
  maxAssetCacheWait: 108 ms
  unsuspendAfter: 181 ms
popupPanel: 
  blocked: 13
  blockedDetails: 
    arsnivyr.com: 2
    faiptauz.com: 1
    googletagmanager.com: 1
    oaphoace.net: 1
    offerimage.com: 4
    rndskittytor.com: 1
    tzegilo.com: 1
    whairtoa.com: 2
```
</details>

**Comment by AntiAdvert** [comment]
- https://github.com/uBlockOrigin/uAssets/issues/15604#issuecomment-1310680654
- `*$xhr,3p,domain=tainio-mania.online`

**Comment by fanis2022** [comment]
thanks it works!

**Comment by fanis2022** [comment]
thanks
   Στις Πέμπτη 10 Νοεμβρίου 2022 - 09:06:57 μ.μ. EET, ο χρήστης mapx ***@***.***> έγραψε:  
 
 


Closed #15611 as completed via c833a24.
